docs/modules/cloudhub/CloudHub.md

```diff
@@ -0,0 +1,57 @@
+# CloudHub
+
+## Overview
+
+Cloud hub is a web socket client, which is mediator between EdgeController and the Edge side.
+It function is to do the communication between edge and the EdgeController.
+
+The connection to the edge(through EdgeHub module) is done through the HTTP over Websocket connection.
+For internal communication it directly communicates with the EdgeController.
+All the request send to CloudHub is of context object which are stored in channelQ along with the
+mapped channels of event object marked to its nodeID.
+
+
+The main functions performed by CloudHub are :-
+
+- Get message context and create ChannelQ for events
+- Create http connection over websocket
+- Serving Websocket connection
```

Comment:
Maintain the tense. 
Serving Websocket - > Serve Websocket connection  
Reading Message - > Read message...